website/project/model.py

```diff
@@ -92,29 +93,40 @@ class MetaSchema(StoredObject):
     # Version of the schema to use (e.g. if questions, responses change)
     schema_version = fields.IntegerField()
 
+    @property
+    def requires_approval(self):
```

Comment:
They aren't used in the existing templates, no - but they're also not tightly coupled to prereg.

I think I disagree on the idea of moving it to `ensure_schema_structure`.